app/src/main/java/com/globant/calculatoronboardingkotlin/mvp/contracts/CalculatorContracts.kt

```diff
@@ -1,19 +1,28 @@
 package com.globant.calculatoronboardingkotlin.mvp.contracts
 
-import com.globant.calculatoronboardingkotlin.mvp.presenter.ErrorMessages
-
 interface CalculatorContracts {
     interface Model {
         var firstNumber: String
         var secondNumber: String
         var operator: String
```

Comment:
pls do not set variables in the contract

Reply:
removed, thanks